operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -221,23 +209,6 @@ Editorial feedback is based on the guidelines laid out in the [Documentation Sty
 
 The [Documentation Style Guide](https://docs.mattermost.com/guides/core.html#documentation-style-guide) and [UI text guidelines](https://docs.mattermost.com/process/documentation-UItext-guidelines.html).
 
-## Customer Education Processes
```

Comment:
@justinegeffen Wondering if we do want to delete this section.  It still seems fairly relevant and I don't want to lose it.

Reply:
I'm happy to keep it. I wasn't sure whether customer education would remain as-is or reworked into something like "Customer Support Processes". 0/5 on renaming. 

Wondering if we could expand on it and incorporate some of the ideas around where and how we manage different types of knowledge as per our discussion.

Reply:
I've added the section back into the doc, @wiersgallak. :)

Reply:
Thanks - It could be Customer Support Processes, but I am 0/5 as well.  I like it as it is.  I'll approve this PR with changes as it is, but agree we could add the other channels of how we get information on new areas we need to build content for.

Reply:
I'm 5/5 on that idea - we can include the #roughdraft and docs process, and forum posts etc. I'll open a new PR for that.